packages/react/src/components/Breadcrumbs/BreadcrumbsList.tsx

```diff
@@ -0,0 +1,42 @@
+import { inner, useMergeRefs } from '@floating-ui/react';
+import { Slot } from '@radix-ui/react-slot';
+import cl from 'clsx/lite';
+import { type HTMLAttributes, forwardRef, useEffect, useRef } from 'react';
+
+export type BreadcrumbsListProps = {
+  /**
+   * Change the default rendered element for the one passed as a child, merging their props and behavior.
+   * @default false
+   */
+  asChild?: boolean;
+} & Omit<HTMLAttributes<HTMLOListElement>, 'size'>;
+
+export const BreadcrumbsList = forwardRef<
+  HTMLOListElement,
+  BreadcrumbsListProps
+>(({ asChild, className, ...rest }, ref) => {
+  const Component = asChild ? Slot : 'ol';
+  const innerRef = useRef<HTMLOListElement>(null);
+  const mergedRefs = useMergeRefs([innerRef, ref]);
+
+  // Set aria-current on last link
+  useEffect(() => {
+    const links = innerRef.current?.querySelectorAll('a') || [];
+    const lastIndex = links?.length - 1;
+
+    links?.forEach((link, index) => {
```

Comment:
Don't use `forEach`. Use [for...of](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of). 

We haven't gotten around to it but we are planning on removing all usages of `foreach` in our codebase in favour of `for...of`.

Some info as to why
- https://biomejs.dev/linter/rules/no-for-each/
- https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/no-array-for-each.md

Reply:
Hm I disagree with this, but will comply 😅